features/format.feature

```diff
@@ -410,10 +410,12 @@ Feature: Custom formats
         """
         And the content of file "2020-08-29_entry-the-first.md" in the cache should be
         """
+        ---
```

Comment:
I think the example in #1065 meant the dashes here as a placeholder for content earlier in the file (not actual dashes).

Reply:
Oh, hey, these dashes actually part of the yaml spec. My mistake!

https://yaml.org/spec/1.2/spec.html#id2760395

Reply:
I think the example was supposed to be actual dashes and dots based on Pandoc documentation that was linked in the ticket. https://pandoc.org/MANUAL.html#metadata-blocks

"A YAML metadata block is a valid YAML object, delimited by a line of three hyphens (---) at the top and a line of three hyphens (---) or three dots (...) at the bottom."